src/DistributedMemcacheMutex.php

```diff
@@ -28,16 +30,28 @@ public function __construct(Memcache $memcache)
      * @param string $name
      * @param int $expirationTimeInSeconds
      *
+     * @throws InvalidArgumentException
      * @throws AlreadyLockedException
      */
     public function lock($name, $expirationTimeInSeconds = self::FIVE_MINUTES_SECONDS)
     {
+        $this->assertServersAddedToMemcache();
```

Comment:
@pies will change to `getVersion()` but I would like to leave it here, because of memcache can be closed somehere in between `Memcache::close()`